admin-core/src/main/kotlin/com/epam/drill/admin/core/WsExtension.kt

```diff
@@ -55,9 +55,9 @@ fun Route.authWebSocket(
 }
 
 private suspend fun DefaultWebSocketServerSession.socketAuthentication(tokenService: TokenService) {
-    val token = call.parameters["token"]
+    val token = call.request.cookies["jwt"] ?: call.parameters["token"]
```

Comment:
I suspect there is an option in WebSocket clients to pass token other than using parameter
_TODO_: investigate and remove this `parameters["token"]` option if true

Reply:
We will need to choose one way to pass the token to the WS connection. An attempt is made here to maintain backward compatibility, because... we didn't discuss the opposite